Makefile

```diff
@@ -5,6 +5,12 @@ ifeq (${PREFIX},)
 	PREFIX := /usr/local
 endif
 
+GOBIN ?= $(shell go env GOPATH)/bin
```

Comment:
```suggestion
GOBIN ?= $(shell go env GOBIN)
```

Reply:
GOBIN could be not set. It is the case for me, for example.

Reply:
The GOBIN environment variable refers to the bin directory inside the go installation directory such as $GOROOT/bin. This variable refers to the location where you can find the binaries. Setting up this environment variable is particularly useful if we frequently switch between different GOPATH directories.

Reply:
> The directory where go install and go get will place binaries after building main packages. Generally this is set to somewhere on the system PATH so that installed binaries can be run and discovered easily.

Reply:
> The GOBIN environment variable refers to the bin directory inside the go installation directory such as $GOROOT/bin.

That is not true. GOBIN is where go install binaries, not where go installation is